easybuild/easyconfigs/i/intel-compilers/intel-compilers-2021.1.eb

```diff
@@ -0,0 +1,21 @@
+name = 'intel-compilers'
+version = '2021.1'
```

Comment:
This should probably be changed to `2021.1.2`, which is the proper version for the compiler components included in HPC Kit 2021.1.0 ...

Reply:
Compiler is 2021.1.1